### PR TITLE
HBASE-22825 [HBCK2] Add a client-side to hbase-operator-tools that ca…

### PR DESCRIPTION
**Pull request by jatsakthi**
…n exploit fixMeta added in server side

### CODE REVIEW

**Comment by jatsakthi** [comment]
Haven't added UTs, as it would be very similar to what we have in TestMetaFixer. Let me know if we would still like it to be present.

**Comment by busbey** [comment]
Do we require the change to bump the HBase 2.1 version to 2.1.6 specifically? If so we'll have to wait for the release.

**Comment by jatsakthi** [comment]
Yes Sean, for availing the fixMeta api we need to.

File: pom.xml [hunk 1]
```diff
@@ -123,7 +123,7 @@
     <compileSource>1.8</compileSource>
     <java.min.version>${compileSource}</java.min.version>
     <maven.min.version>3.3.3</maven.min.version>
-    <hbase.version>2.1.2</hbase.version>
+    <hbase.version>2.1.6-SNAPSHOT</hbase.version>
```
Comment (busbey) [inline pom.xml]: 2.1.6 is out now.

**Comment by saintstack** [comment]
Let me close this one and make a new one w/ @jatsakthi 's patch amended referring to released 2.1.6.

**Comment by busbey** [comment]
Sounds good. Next time you should be able to push a commit into this PR with the amendment you want, fyi.

**Pull request closed without merging**